src/client.ts

```diff
@@ -125,12 +88,26 @@ export class ClasschartsClient {
     options?.to && params.append("to", options?.to);
     options?.last_id && params.append("last_id", options?.last_id);
     return this.makeAuthedRequest(
-      API_BASE + "/activity/" + this.studentId + "?" + params.toString(),
+      this.API_BASE + "/activity/" + this.studentId + "?" + params.toString(),
       {
         method: "GET",
       }
     );
   }
+
+
+  async getPupil(options?: GetActivityOptions): Promise<ActivityResponse> {
```

Comment:
Not exactly sure what this line does as I can't test this on a parent account myself but it seems to error if used by the student client. Perhaps it could be moved to the parent client?
(Also types might need to be updated for both options and response)